src/components/layer-controller/ChannelController.js

```diff
@@ -9,6 +9,12 @@ import debounce from 'lodash/debounce';
 import isEqual from 'lodash/isEqual';
 
 import ChannelOptions from './ChannelOptions';
+import { getSourceFromLoader } from '../../utils';
+
+// min is not exported.
+DTYPE_VALUES.Int8.min = -((DTYPE_VALUES.Uint8.max + 1) / 2);
+DTYPE_VALUES.Int16.min = -((DTYPE_VALUES.Uint16.max + 1) / 2);
+DTYPE_VALUES.Int32.min = -((DTYPE_VALUES.Uint32.max + 1) / 2);
```

Comment:
The `Uint` constants also don't have `min`. So is `min` undefined below when dtype is unsigned?

Reply:
it makes me a little uncomfortable to be mutating this import. I would be fine with defining a new constant with just the domain mappings:

```javascript
const DOMAINS = {
   Uint8: [0, DTYPE_VALUES.Uint8.max],
   /* ... */
}
```

Reply:
I can take responsibility for not having `min` defined in Viv. It was a suggested from @ilan-gold recently and I pushed back. My primary motivation was to decrease the API surface area for Viv. 

I think my main hesitation is that `DTYPE_VALUES` is an internal constant with _lots_ of fields specific to webgl. Exposing this as a part of Viv's public API, means that folks may start to rely on parts of API in ways we don't anticipate/intend (as shown here with `max`). This means we are less flexible in the future to make changes to Viv's internals because we've exposed the internals to library users.

In my opinion, adding a field like `min` would be addition that in intended _specifically_ for the public usage of the constant which feels odd. With that said, I think it totally makes sense to have a function in Viv to get `domain` for a dtype, but in that case I think we should expose a function with that exact behavior rather than letting others choose what information they want to access from `DTYPE_VALUES`:

```javascript
import { getDomain } from '@hms-dmbi/viv';

const [min, max] = getDomain(pixelSource); // can define min/max mapping however we want internally and can change
```

Reply:
I just checked, and we have marked `DTYPE_VALUES` as `@deprecated` in Viv (to be removed in a future release):

https://github.com/hms-dbmi/viv/blob/ab571893952f81193edc02194a03954b81d18f05/src/constants.ts#L13-L83

So currently in VSCode, you should get ~`DTYPE_VALUES`~ when using this constant, including a comment for our motivation.